src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -99,59 +123,97 @@
-  (let [selected-networks         (rf/sub [:wallet/wallet-send-selected-networks])
-        loading-networks          (find-affordable-networks token input-value selected-networks)
+  [{:keys [from-values-by-chain to-values-by-chain routes token theme fetch-routes
+           affordable-networks disabled-from-networks on-press-from-network on-press-to-network]}]
+  (let [token-symbol              (:symbol token)
         loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
-        data                      (if loading-suggested-routes? loading-networks routes)]
-    (if (or (and (not-empty loading-networks) loading-suggested-routes?) (not-empty routes))
-      [rn/flat-list
-       {:data                    (if (and (< (count data) 3) (pos? (count data)))
-                                   (concat data [{:status :add}])
-                                   data)
-        :content-container-style style/routes-container
-        :header                  [rn/view {:style style/routes-header-container}
-                                  [quo/section-label
-                                   {:section         (i18n/label :t/from-label)
-                                    :container-style (style/section-label 0)}]
-                                  [quo/section-label
-                                   {:section         (i18n/label :t/to-label)
-                                    :container-style (style/section-label 64)}]]
-        :render-fn               (fn [item]
-                                   [route-item
-                                    {:amount       amount
-                                     :theme        theme
-                                     :fetch-routes fetch-routes
-                                     :status       (cond
-                                                     (= (:status item) :add)   :add
-                                                     loading-suggested-routes? :loading
-                                                     :else                     :default)
-                                     :from-network (if loading-suggested-routes?
-                                                     (utils/id->network item)
-                                                     (utils/id->network (get-in item [:from :chain-id])))
-                                     :to-network   (if loading-suggested-routes?
-                                                     (utils/id->network item)
-                                                     (utils/id->network (get-in item
-                                                                                [:to :chain-id])))}])}]
+        data                      (if loading-suggested-routes? affordable-networks routes)]
+    (if (or (and (not-empty affordable-networks) loading-suggested-routes?) (not-empty routes))
+      (let [data (if (not-empty disabled-from-networks)
```

Comment:
In general I find a lot of the code in these send files quite difficult to parse because there is so many things going on. Mostly that's due to the complex nature of everything involved with the send related code, however I think we should do our best to make the code as readable as possible so that it is easier for someone else to maintain/ add to in the future.